code/game/machinery/computer/arcade.dm

```diff
@@ -407,6 +416,16 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	var/gameStatus = ORION_STATUS_START
 	var/canContinueEvent = 0
 
+	var/obj/item/radio/Radio
+	var/list/gamers = list()
```

Comment:
This should be a static list so the warnings persist across multiple machines

Reply:
Not having it static means it will only work on roundstart machines, not player created ones?

Reply:
having it static will work on all machines, player created or roundstart

Reply:
not having it static will do the exact same thing but it will not be shared between all machines